service/order_service.py

```
@@ -83,21 +96,88 @@ async def handle_chat(
+
+    def _schedule_prefetch(self, session_id: int, suggestions: list[str], mode: str) -> None:
+        """suggestions 중 프리패치 적합한 것만 백그라운드 태스크로 실행한다.
+
+        실시간 상태에 의존하는 항목(장바구니·결제·주문 변경)은 캐싱해도 stale해지므로 건너뛴다.
+        """
+        for text in suggestions:
+            if _is_prefetchable(text):
+                asyncio.create_task(
+                    self._run_prefetch(session_id, text, mode),
+                    name=f"prefetch-{session_id}",
+                )
```

Ruff (0.15.12)

[warning] Store a reference to the return value of `asyncio.create_task`

(RUF006)